tests/backends/storage/test_fs.py

```diff
@@ -0,0 +1,102 @@
+"""Tests for Ralph fs storage backend"""
+
+from collections.abc import Iterable
+from pathlib import Path
+
+import pytest
+
+from ralph.backends.storage.fs import FSStorage
+from ralph.defaults import APP_DIR, FS_STORAGE_DEFAULT_PATH
+
+
+# pylint: disable=invalid-name
+# pylint: disable=unused-argument
+def test_fs_storage_instanciation(fs):
+    """Test the FSStorage backend instanciation"""
+    # pylint: disable=protected-access
+
+    assert FSStorage.name == "fs"
+
+    storage = FSStorage()
+
+    assert storage._path == Path(FS_STORAGE_DEFAULT_PATH)
+
+    deep_path = "deep/directories/path"
+
+    storage = FSStorage(deep_path)
+
+    assert storage._path == Path(deep_path)
```

Comment:
You should also assert that it has been created! And check that it does not raise an error if this path already exists.